### ISSUE TITLE
Expo broken dependency

### ISSUE THREAD

**Issue by butri71** [echowaves/expo-storage#1]
Can you please confirm if this package is compatible with Expo SDK 40?

I'm getting the following warning:

Could not resolve dependency:
npm ERR! peer expo@"^42.0.3" from expo-storage@1.0.1
npm ERR! node_modules/expo-storage
npm ERR!   expo-storage@"^1.0.1" from the root project

Thanks

**Comment by dmitryame** [comment]
I just published version 1.0.2. 
Apparently, there is no explicit dependency on expo 42.0.3 in this little npm module (it's really simple). As such I removed it from the package.json and published new version. Not sure if "expo-file-system": "~11.1.3" is backwards compatible with the earlier versions of expo, but give it a try -- hope it works for you.

**Comment by butri71** [comment]
I've downloaded your latest version, thank you for that!

It seems to be working ok and the warning seems to be gone.

I have the usual warning about missing declaration though around the import, nothing major I guess:

```
Could not find a declaration file for module 'expo-storage'. '/Users/maxaquilino/Courses/React Native/cocktail/node_modules/expo-storage/lib/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/expo-storage` if it exists or add a new declaration (.d.ts) file containing `declare module 'expo-storage';`ts(7016)

```

Currently I'm using the storage for a 6MB file, I understand from your post that there is no space limitation since it just uses expo-file-system to save the file somewhere in the device.

Anyway, congratulation for this package, it's awesome!

Cheers
